### ISSUE TITLE
luadata error prevents 'wear' option serverside

### ISSUE THREAD

**Issue by VinylStorm** [CapsAdmin/pac3#307]
I'm aware there is a previous topic to this, but it seems that ours varies under the circumstances.

To begin, we have a NutScript server. PAC3 is used alot, but we never experienced it breaking or becoming disabled before. Recently, the server crashed and PAC3's wear option didnt work upon restarting the server. It took a manual restart to fix PAC.

Recently, hard restarts have stopped working. PAC's wear option will not load anyone's PAC. They can wear it clientside, but it wont appear serverside.

I get an error when opening the PAC editor, but apparently others do not get this. My confusion is that it is that no one can use wear, at all.

My error opening the editor is:
Decoding pac3//Slug.txt failed: luadata_decode:1: unexpected symbol near '\'
When I click 'wear':
[PAC] Transmitting outfit (2.25 KB)
But nothing loads or anything comes after that.

There was no concrete evidence behind what is causing the problem or how to fix it. If anyone can figure this out, that would be great.

Server is up to date with NutScript.
Average pop: 12 players

**Comment by LUModder** [comment]
Not a server issue, issue with the file

**Comment by VinylStorm** [comment]
The problem is nobody can get any of their PAC's loaded.
This problem has recently begun.

We have a lot of content mounted, if that means anything. 
And if its not PAC, then it has to be our server. Where could we look for the source of the problem in the server files?

**Comment by LUModder** [comment]
Then something in nutscript is escaping strings improperly, thus why I
don't even touch the gamemode because of how hacky things are.

tl;dr don't use pac with nutscript unless you have developer knowledge with
both
